src/hbbft.erl

```diff
@@ -93,10 +108,61 @@ handle_msg(Data = #data{round=R}, J, {dec, R, I, Share}) ->
             %% not enough shares yet
             {Data#data{dec_shares=NewShares}, ok}
     end;
+handle_msg(Data = #data{round=R}, J, {sign, R, BinShare}) ->
```

Comment:
This is not part of the protocol specification as far as I can tell. Perhaps a bit of an explanation will be good. I suspect this happens after step#3 in the hbbft protocol?

![image](https://user-images.githubusercontent.com/3199183/38527339-11be0af0-3c0f-11e8-9325-0d78af463d99.png)